src/app/shared/formly/extensions/translate-select-options.extension.ts

```diff
@@ -4,26 +4,26 @@ import { isObservable, of } from 'rxjs';
 import { map, startWith, tap } from 'rxjs/operators';
 
 /**
- * Extension to translate the templateOptions.options and add a placeholder element.
+ * Extension to translate the props.options and add a placeholder element.
  *
- * @templateOption  **options**  defines options to be shown as key value pairs. Accepts two types:
+ * @props  **options**  defines options to be shown as key value pairs. Accepts two types:
  * * `` { value: any; label: string}[]``
  * * `` Observable<{ value: any; label: string}[]>``
- * @templateOption **placeholder** - is used to add a placeholder element. This will also be translated.
+ * @props **placeholder** - is used to add a placeholder element. This will also be translated.
  *
  * @usageNotes
  * It will use the TranslateService to translate option labels.
- * These modified options are written to ``templateOptions.processedOptions``.
+ * These modified options are written to ``props.processedOptions``.
  */
 class TranslateSelectOptionsExtension implements FormlyExtension {
   constructor(private translate: TranslateService) {}
 
   prePopulate(field: FormlyFieldConfig): void {
-    const to = field.templateOptions;
+    const to = field.props;
```

Comment:
maybe we should also use props as variable name here - there are other occurrences of this issue, too